src/System.Security.Cryptography.Algorithms/src/System.Security.Cryptography.Algorithms.csproj

```diff
@@ -6,8 +6,8 @@
     <OutputType>Library</OutputType>
     <AssemblyName>System.Security.Cryptography.Algorithms</AssemblyName>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <DefineConstants>INTERNAL_ASYMMETRIC_IMPLEMENTATIONS</DefineConstants>
-    <NoWarn>CA5351;$(NoWarn)</NoWarn>
+    <DefineConstants>$(DefineConstants);INTERNAL_ASYMMETRIC_IMPLEMENTATIONS</DefineConstants>
+    <NoWarn>CS3016;CA5351;$(NoWarn)</NoWarn>
```

Comment:
As it stands, this library is paying for stackalloc's to be cleared.  This change adds a bunch of them, so consider adding:
https://github.com/dotnet/corefx/blob/bece3a206ec9fccd9a425447ac49a36bfb0d1bb0/src/System.Net.Sockets/src/System.Net.Sockets.csproj#L10

Reply:
To avoid potential scope creep I'll add ILLinkClearInitLocals in a followup change.